Marlin/src/inc/Conditionals_post.h

```diff
@@ -2101,11 +2101,9 @@
     #define PREHEAT_COUNT 3
   #elif defined(PREHEAT_2_LABEL)
     #define PREHEAT_COUNT 2
-  #else
+  #elif defined(PREHEAT_1_LABEL)
```

Comment:
This should not be that way...lavel 1 check is already done above

Reply:
ok I saw now...just remove on fisrt if. I'm trying to add same logic to all mesh preheats